libs/ha/ha_common_libs_k8s.py

```diff
@@ -1480,6 +1482,9 @@ def delete_kpod_with_shutdown_methods(self, master_node_obj, health_obj,
             remaining.extend(master_node_obj.get_all_pods(pod_prefix=ptype))
 
         LOGGER.info("Delete %s by %s method", delete_pods, down_method)
+        if event is not None:
+            LOGGER.debug("Setting the Thread event")
+            event.set()
```

Comment:
We need to add event.set() inside loop as we need to clear event inside loop only because of status check.